pkg/publicapi/endpoint/shared/endpoint.go

```diff
@@ -78,6 +77,14 @@ func (e *Endpoint) nodeInfo(c echo.Context) error {
 	return c.JSON(http.StatusOK, e.nodeStateProvider.GetNodeState(c.Request().Context()))
 }
 
+type VersionRequest struct {
+	ClientID string `json:"client_id" example:"ac13188e93c97a9c2e7cf8e86c7313156a73436036f30da1ececc2ce79f9ea51"`
+}
+
+type VersionResponse struct {
+	VersionInfo *models.BuildVersionInfo `json:"build_version_info"`
+}
+
```

Comment:
lets move all api models to `publicapi/apimodels`

Reply:
actually we already have `apimodels.GetVersionResponse` and I think we can deprecate most of the APIs under `endpoint.shared`, specifically `node_info` and `version`

Reply:
and lets not use `ClientID` if we can

Reply:
actually it seems we need `/` to return the node version for better user experience. Lets maintain that behaviour, but avoid code duplication